hbase/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -92,6 +94,9 @@
      */
     public boolean _clientSideBuffering = false;
     public long _writeBufferSize = 1024 * 1024 * 12;
+    public int multiGetsCount = 0;
```

Comment:
Thanks for the reviews. I can change the name. 
Yes Sean, it is the number that indicates the count of the total number of gets that will be issued in one RPC call.